content/general/presence.textile

```diff
@@ -0,0 +1,21 @@
+---
+title: Presence
+section: general
+index: 27
+---
+
+Presence allows you to know the connection status of a client or a device. Apart from letting you know who is online/offline at a particular point in time, you can also subscribe to presence updates( Note: This is only available with Ably's "Realtime Library":/realtime ). This means that Ably will send a message to all it's presence subscribers whenever a client on the same presence channel that they are subscribed to, has left or a new client has joined.
```

Comment:
I think replace this with:

Ably's presence feature allows devices to announce their presence on a channel.  Other devices or services may then subscribe to these presence events (such as entering, updating their state, or leaving the channel) in real time using our realtime SDKs, or via the Reactor service.  You can also request a list of who is online/offline on a channel at a particular point in time via the REST API.

Reply:
s/devices/clients I would suggest